src/shared/domain/definition/case-field.model.ts

```diff
@@ -26,4 +26,5 @@ export class CaseField implements Orderable {
 
   @Type(() => WizardPageField)
   wizardProps?: WizardPageField;
+  metadata?: boolean;
```

Comment:
Why this as part of this story?

Reply:
I needed to skip metadata fields from being registered to formGroup. Metadata fields are actually coming through as part of caseFields